src/components/CompensationCalculator/compensation_data/sf_benchmark.ts

```diff
@@ -1,10 +1,10 @@
 export const sfBenchmark: Record<string, number> = {
+    'Community Engineer': 199000,
     'Content Marketer': 165000,
     'Customer Success Manager': 192000,
     'Front End Developer': 199000,
     'Full Stack Engineer': 216000,
     'Graphic Designer': 128400,
-    'Growth Engineer': 192000,
```

Comment:
Ah I've been meaning to talk about this. I think we should go back to advertising for growth engineer instead of full stack, as most candidates I've talked to all have no growth experience.

Reply:
Ok cool - will do a refresh of the spec then on Monday, I think it needs a bit of improving generally anyway